### ISSUE TITLE
No results found

### ISSUE THREAD

**Issue by ilyaliao** [marconi/switch-notebook#11]
Hi! 
Thanks for making the great plugin.
It is really helpful for me.

I have encountered an issue where, upon opening the plugin, I am unable to locate my notebooks. 
Please see the picture for reference.

![截圖 2023-12-21 上午11 18 03](https://github.com/marconi/switch-notebook/assets/93901409/f72849c2-5e5a-49f2-917a-cd32f93f1581)

I tried to identify the issue and found that this piece of code is not successfully storing the books.

```js
inkdrop.onAppReady(() => {
  const { books } = inkdrop.store.getState();
  const options = books.all.map(({ _id, name }) => ({
    key: _id,
    value: _id,
    text: name,
  }));
  this.setState({ options });
});
```

I attempted to modify it like this, and it temporarily resolved the issue. Currently, it is running successfully. 
I'm not sure if I'm the only one experiencing this problem.

```js
buildDictionary() {
  const { books } = inkdrop.store.getState();
  const options = books.all.map(({ _id, name }) => ({
    key: _id,
    value: _id,
    text: name,
  }));
  this.setState({ options });
}

toggle() {
  if (!this.dialogRef.current.isShown) {
    this.buildDictionary();
    this.dialogRef.current.showDialog();
  } else {
    this.dialogRef.current.dismissDialog();
  }
}
```

**Comment by marconi** [comment]
@CofCat456 Thanks for flagging, if you can open a PR I'll be happy to review and test it. 😄

**Comment by marconi** [comment]
@CofCat456 Alright just pushed and published version `0.2.3`. 🎉 🌮
